File: 백준/Silver/20114. 미아 노트/미아 노트.py
import sys
input = sys.stdin.readline

n, h, w = map(int, input().rstrip().split())
result = ['?' for _ in range(n)]
for _ in range(h):
    string = input().rstrip()

    for i in range(n):
        for j in range(w):
            if string[i*w+j] != '?':
                result[i] = string[i*w+j]

print(''.join(result))

File: 백준/Silver/20114. 미아 노트/미아 노트.cc
#include <stdio.h>

char str[15][1005];
char result[105];
int main(void) {
    int n, h, w;
    scanf("%d %d %d", &n, &h, &w);
    for (int i = 0; i < h; i++) scanf("%s", str[i]);
    for (int i = 0; i < n; i++) result[i] = '?';
    result[n] = 0;

    for (int i = 0; i < h; i++) {
        for (int j = 0; j < n * w; j++) {
            if (str[i][j] != '?') result[j/w] = str[i][j];
        }
    }

    printf("%s", result);
}